plugin/trino-vertica/src/test/java/io/trino/plugin/vertica/TestVerticaConnectorTest.java

```diff
@@ -0,0 +1,254 @@
+                .row("clerk", "varchar(15)", "", "")
+                .row("shippriority", "bigint", "", "")
+                .row("comment", "varchar(79)", "", "")
+                .build();
+        MaterializedResult actualColumns = computeActual("DESCRIBE orders");
+        assertEquals(actualColumns, expectedColumns);
+    }
+
+    @Override
+    public void testShowCreateTable()
+    {
+        // shippriority column's type is BIGINT, not INTEGER
+        assertThat(computeActual("SHOW CREATE TABLE orders").getOnlyValue())
+                .isEqualTo("CREATE TABLE vertica.tpch.orders (\n" +
+                        "   orderkey bigint,\n" +
+                        "   custkey bigint,\n" +
+                        "   orderstatus varchar(1),\n" +
+                        "   totalprice double,\n" +
+                        "   orderdate date,\n" +
+                        "   orderpriority varchar(15),\n" +
+                        "   clerk varchar(15),\n" +
+                        "   shippriority bigint,\n" +
+                        "   comment varchar(79)\n" +
+                        ")");
+    }
+
+    @Test
+    @Override
+    public void testShowColumns()
+    {
+        // shippriority column's type is BIGINT, not INTEGER
+        MaterializedResult actual = computeActual("SHOW COLUMNS FROM orders");
+        MaterializedResult expected = resultBuilder(getSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
+                .row("orderkey", "bigint", "", "")
+                .row("custkey", "bigint", "", "")
+                .row("orderstatus", "varchar(1)", "", "")
+                .row("totalprice", "double", "", "")
+                .row("orderdate", "date", "", "")
+                .row("orderpriority", "varchar(15)", "", "")
+                .row("clerk", "varchar(15)", "", "")
+                .row("shippriority", "bigint", "", "")
+                .row("comment", "varchar(79)", "", "")
+                .build();
+        assertEquals(actual, expected);
+    }
+
+    @Test
+    @Override
+    public void testDropColumn()
+    {
+        String tableName = "test_drop_column_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT 123 x, 456 y, 111 a", 1);
+
+        assertQueryFails("ALTER TABLE " + tableName + " DROP COLUMN x", "This connector does not support dropping columns");
+
+        assertUpdate("DROP TABLE " + tableName);
+    }
+
+    @Override
+    protected Optional<DataMappingTestSetup> filterDataMappingSmokeTestData(DataMappingTestSetup dataMappingTestSetup)
+    {
+        String typeName = dataMappingTestSetup.getTrinoTypeName();
+        if (typeName.equals("time")
+                || typeName.equals("timestamp")
+                || typeName.equals("timestamp(3) with time zone")
+                || typeName.equals("varbinary")) {
+            return Optional.of(dataMappingTestSetup.asUnsupported());
+        }
+
+        if (typeName.equals("real")
```

Comment:
Why does this fail even with the type mapping implemented? What's the error?

Reply:
Here is the error and the cause. Let me check if we should remove support for `real` in the initial PR.
```bash
java.lang.AssertionError: For query: 
 SELECT row_id FROM test_data_mapping_smoke_real_ocehj WHERE value = REAL '567.123'
not equal
Actual rows (up to 100 of 0 extra rows shown, 0 rows in total):
    
Expected rows (up to 100 of 1 missing rows shown, 1 rows in total):
    [sample value]
```

```bash
/# /opt/vertica/bin/vsql -hlocalhost -Utest_user -wtest_password
test_user=> select * from tpch.test_real;
        c1
------------------
 567.122985839844
```